contracts/script/deploy/DeployL2T1XChainReader.s.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
I suggest to create a 7683 folder to differentiate the protocol deployments from intent-related contracts.

Reply:
done